slicingTool/runProjectSlicingTool.sh

```diff
@@ -35,13 +35,17 @@ then
     if [[ $USER =~ "curator" ]]
     then
         # Attempts to find and use database $DB and if successful it is backed up
-        # before being dropped
-        if mysql -u $USER -p$PASS -e "use $DB";
+        # before being dropped (errors, if any, stored in $DB_ERROR)
+        if DB_ERROR=$(mysql -u $USER -p$PASS -e "use $DB" 2>&1 > /dev/null);
```

Comment:
Still executing the statement in the `if`... could this be moved to _before_ the if? I don't find this any more readable than before.

Reply:
Done!